### PR TITLE
Remove core-js from dependencies

### PR DESCRIPTION
**Pull request by znarf**
Seems it's not used anymore.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,6 @@
   },
   "dependencies": {
     "classnames": "~2.2.6",
-    "core-js": "~2.6.5",
```
Comment (ad1992) [inline package.json:40]: can you also update the lock file?

Comment (znarf) [reply package.json:40]: @ad1992 usually `package-lock.json` should not be committed for libraries, so I suggest to delete it instead.

Comment (znarf) [reply package.json:40]: @ad1992 updated it anyway in case that was a blocker

Comment (ad1992) [reply package.json:40]: we commit package.json so that all the contributors have the same set up in local development

**Comment by ad1992** [comment]
@znarf thanks for your contribution.

**Pull request merged**